src/modules/package.rs

```diff
@@ -102,11 +102,19 @@ fn get_setup_cfg_version(context: &Context, config: &PackageConfig) -> Option<St
 }
 
 fn get_gradle_version(context: &Context, config: &PackageConfig) -> Option<String> {
-    let file_contents = context.read_file_from_pwd("build.gradle")?;
-    let re = Regex::new(r#"(?m)^version ['"](?P<version>[^'"]+)['"]$"#).unwrap();
-    let caps = re.captures(&file_contents)?;
-
-    format_version(&caps["version"], config.version_format)
+    context
+        .read_file_from_pwd("build.gradle")
+        .and_then(|contents| {
+            let re = Regex::new(r#"(?m)^version ['"](?P<version>[^'"]+)['"]$"#).unwrap(); /*dark magic*/
+            let caps = re.captures(&contents)?;
+            format_version(&caps["version"], config.version_format)
+        })
+        .or_else(|| {
+            let properties_file_contents = context.read_file_from_pwd("gradle.properties")?;
+            let re = Regex::new(r"version=(?P<version>.*)").unwrap();
+            let caps = re.captures(&properties_file_contents)?;
+            format_version(&caps["version"], config.version_format)
```

Comment:
Given the comments from [moritzreiter](https://github.com/moritzreiter), please read the properties file first.